Formula/php.rb

```diff
@@ -0,0 +1,344 @@
+class Php < Formula
+  desc "General-purpose scripting language"
+  homepage "https://php.net"
+  url "https://github.com/php/php-src/archive/php-7.1.8.tar.gz"
+  sha256 "f3be4262203fa9db3d126832a1632e9a80e52e313f4230218c392d6a9178c368"
+
+  # javian: not sure about the origin of this so I'll keep it commented for now. Could it have something to do with building extensions ?
+  # javian: This has been in the formula for 8+ years but have found no proof that it is needed anymore
+  # So PHP extensions don't report missing symbols
+  # skip_clean "bin", "sbin"
+
+  # TODO
+  # Opcache default config file is included in extension formula, is it needed ?
+  # ldap extension: I have a vague recollection that this caused some issues that were resolved with exluding it. needs to be checked.
+  # Need to add tests for all supported SAPIs
+
+  option "with-imap-uw", "Build PHP IMAP extension"
+  option "with-thread-safety", "Build with thread safety"
+
+  # javian: Not yet checked these options
+  option "with-pdo-oci", "Include Oracle databases (requries ORACLE_HOME to be set)"
+  option "without-ldap", "Build without LDAP support"
+  option "without-unixodbc", "Build without unixODBC support"
+
+  depends_on "autoconf" => :build
```

Comment:
Only necessary for head build.

Reply:
Ah, good point. I assumed we were using the php.net tarball. I can't recall what all exactly happens in their release process, other than running the build tools so they aren't needed downstream.